src/components/Questions/sampleData.js

```diff
@@ -0,0 +1,907 @@
+          "answers": {}
+      },
+      {
+          "question_id": 542908,
+          "question_body": "gg",
+          "question_date": "2021-11-04T00:00:00.000Z",
+          "asker_name": "gg",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542896,
+          "question_body": "gg",
+          "question_date": "2021-11-02T00:00:00.000Z",
+          "asker_name": "gg",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542855,
+          "question_body": "g",
+          "question_date": "2021-11-01T00:00:00.000Z",
+          "asker_name": "g",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542854,
+          "question_body": "g",
+          "question_date": "2021-11-01T00:00:00.000Z",
+          "asker_name": "g",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542853,
+          "question_body": "test",
+          "question_date": "2021-11-01T00:00:00.000Z",
+          "asker_name": "gg",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542852,
+          "question_body": "Why am i testing this",
+          "question_date": "2021-11-01T00:00:00.000Z",
+          "asker_name": "solanaceae",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542788,
+          "question_body": "Why am i testing this",
+          "question_date": "2021-10-29T00:00:00.000Z",
+          "asker_name": "solanaceae",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542787,
+          "question_body": "Why am i testing this",
+          "question_date": "2021-10-29T00:00:00.000Z",
+          "asker_name": "solanaceae",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542786,
+          "question_body": "Why am i testing this",
+          "question_date": "2021-10-29T00:00:00.000Z",
+          "asker_name": "solanaceae",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      },
+      {
+          "question_id": 542785,
+          "question_body": "Why am i testing this",
+          "question_date": "2021-10-29T00:00:00.000Z",
+          "asker_name": "solanaceae",
+          "question_helpfulness": 0,
+          "reported": false,
+          "answers": {}
+      }
+  ]
+}
+
+
+// var testing = () => {
+//   sampleData.results.forEach(question => {
+//     console.log(question.question_body)
+//   })
+// }
+
+debugger;
```

Comment:
I'm not sure if lines 882 - 905 need to be in the main code. Looks like you were using them for testing. Could you elaborate?

Reply:
No you're right, got trigger happy. I'll see if I can adjust in that branch and push it up, tbd.

Reply:
@rvtaylor94 My brain is done for the night lol. Should be good now.